File: MyVidu1_Intent/app/src/main/java/ntu/ngonguyenanhtu63135957/myvidu1_intent/MainActivity.java
package ntu.ngonguyenanhtu63135957.myvidu1_intent;

import android.os.Bundle;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void ChuyenManHinh(View view) {
        Intent iManHinhKhac = new Intent(MainActivity.this, SubActivityOne.class);
        startActivity(iManHinhKhac);
    }

}

File: MyVidu1_Intent/app/src/main/java/ntu/ngonguyenanhtu63135957/myvidu1_intent/MainActivity.kt
package ntu.ngonguyenanhtu63135957.myvidu1_intent

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import ntu.ngonguyenanhtu63135957.myvidu1_intent.ui.theme.MyVidu1_IntentTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            MyVidu1_IntentTheme {
                // A surface container using the 'background' color from the theme
                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background) {
                    Greeting("Android")
                }
            }
        }
    }
}

@Composable
fun Greeting(name: String, modifier: Modifier = Modifier) {
    Text(
            text = "Hello $name!",
            modifier = modifier
    )
}

@Preview(showBackground = true)
@Composable
fun GreetingPreview() {
    MyVidu1_IntentTheme {
        Greeting("Android")
    }
}